interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1357,18 +1357,21 @@ def get_yaml(self, out):
 
 class table:
     """User provided thermo table for BinarySolutionTabulatedThermo"""
-    def __init__(self, moleFraction=([],''), enthalpy=([],''), entropy=([],'')):
+    def __init__(self, moleFraction=([],''), enthalpy=([],''), entropy=([],''), partialMolarVolume=([],'')):
```

Comment:
Since the CTI/XML formats are going to be removed in the next few versions, we are preferring to add new features only to the YAML format. Unless there is already a significant userbase for this feature who are using CTI/XML files, we'd prefer to not change any of the converters.